dmref_analyzer/RegressionModel.py

```diff
@@ -35,7 +34,7 @@ def gen_x(rng=np.arange(17, 10000)):
     tList = []
     dList = []
     gasAtmList = []
-    for root, dirs, filenames in os.walk('../../Ji Hao/'):
+    for root, dirs, filenames in os.walk('./RamanData/'):
```

Comment:
When you call genDendro.py from within the tests directory, "./RamanData" won't exist. 

I think here we can use "__file__" to pinpoint where the RamanData directory is.

There are two other occurrences of this code at line 85 and line 110

Reply:
What do you mean by "file"?
we can use >>> os.path.join(os.getcwd(), 'RamanData')

Right?